weave

```diff
@@ -108,6 +108,7 @@ exec_remote() {
         -e WEAVE_CONTAINER_NAME \
         -e WEAVE_MTU \
         -e WEAVE_NO_FASTDP \
+        -e WEAVE_NO_BRIDGED_FASTDP \
```

Comment:
The main reason I included this was to lay groundwork for #1681, but it was also useful in comparative performance testing.